guides/Scripting_Languages.md

```diff
@@ -0,0 +1,320 @@
+
+Setup in conf.lua:
+
+``` lua
+-- This allows us to have a dedicated directory for Lua c libraries.
+local os = lovr.system.getOS()
+if os == 'Linux' then
+    package.cpath = 'utils/lua_clib/?.so'
+end
+if os == 'Windows' then
+    package.cpath = 'utils/lua_clib/?.dll'
+end
+if os == 'macOS' then
+    package.cpath = 'utils/lua_clib/?.so'
+end
+
+-- Adjust the require search paths to your needs.
+local lua_path = './?.lua;./?/init.lua;utils/moonscript/?/init.lua;'
+lovr.filesystem.setRequirePath(lua_path)
+
+-- Moonscript must be located at utils. Check for utils/moonscript/moonscript/init.lua.
+-- The LPeg library is to be located at:
+-- Linux:    utils/lua_clib/lpeg.so
+-- Windows:  utils\lua_clib\lpeg.dll
+-- MacOS:    utils/lua_clib/lpeg.so
+-- fuse mode:
+-- Linux and Windows: Beside the fused binary or in the save directory top-level.
+-- MacOS: In the save directory top-level
+
+-- Moonscript rewrites and sets its require paths from package.path when first required.
+-- If you want to alter it afterwards redefine package.moonpath.
+package.path = lua_path
+local moonscript = require'moonscript'
+-- Disable the conventional lua require path for easier testing if fuse mode works.
+-- Every require relies on lovr.filesystem instead.
+package.path = ''
+
+-- This loader uses lovr.filesystem instead of io.open.
+-- Enables require from LOVR's save directory.
+-- Fuse mode won't work at all without it.
+local moon_loader = function(name)
+    local name_path = name:gsub("%.", "/")
+    local file, file_path
+    for path in package.moonpath:gmatch("[^;]+") do
+        file_path = path:gsub("?", name_path)
+        file = lovr.filesystem.read(file_path)
+        if file then break end
+    end
+    if file then
+        local res, err = moonscript.loadstring(file, "@" .. file_path)
+        if not res then
+            error(file_path .. ": " .. err)
+        end
+        return res
+    end
+    return nil, "Could not find moon file"
+end
+moonscript.remove_loader()
+local loaders = package.loaders or package.searchers
+table.insert(loaders, moon_loader)
+
+-- This will require the first .lua or .moon file matching the require paths.
+-- If both are present the Lua file is required.
+require'myConf'
+```
+
+The main.lua file:
+
+``` lua
+-- Requires either .moon or .lua module files.
+-- If both are present the Lua one is required.
+require'myMain'
+```
+
+(TODO: Write about error line translation.)
+
+Wu (Rust dialect)
+---
+
+<img align="right" width="300" height="300" src="https://preview.ibb.co/ePa1eH/wu_dragon.png" alt="Wu Logo">
+
+* Home: [Wu Guide](https://wu-lang.gitbook.io/guide) 
+* Source: [github](https://github.com/wu-lang/wu)
+* Syntax: [Rust](https://rust-lang.org) inspired
+* Semantic: (TODO)
+* jit-compiler: Not available
+* Dependencies: Rust
+* Features
+    * Gradual typing
+    * Rust-like trait system
+    * Expression oriented design
+    * Pattern matching
+    * Cool logo
+
+#### ahead-of-time
+
+(TODO)
```

Comment:
And here are the Wu bindings I know about https://github.com/darltrash/lowr

Reply:
Same as obove.

Reply:
I have added this and the one above.